File: Python/Functions/Fibonacci.py
def fibo(n):
    a = 0
    b = 1
    if n == 0:
        print(0)
        return
    if n == 1:
        print(1)
        return 
    c = 0
    for i in range(2,n+1):
        c = a + b
        a = b
        b = c

    print(c)

fibo(0)
fibo(1)
fibo(2)
fibo(3)
fibo(4)
fibo(5)
fibo(6)
fibo(7)

File: Java/GFG Problems/Dynamic Programming/Fibonacci.java
public class Fibonacci{
    public static void main(String[] args) {
        int n = 999;
        System.out.println(Fibo(n));
    }
    static long Fibo(int n){
        if(n <= 0){
            return 0;
        }
        if(n <= 2){
            return 1;
        }
        long a = 0;
        long b = 1;
        long c = 0;
        for(int i = 2;i<=n;i++){
            c = a + b;
            a = b;
            b = c;
        }
        return c;
        // return c % ((int)(Math.pow(10,8))+7);
    }
}